```
lit.py: /home/build-user/llvm-project/llvm/utils/lit/lit/TestingConfig.py:138: fatal: unable to parse config file '/home/build-user/swift/test/lit.cfg', traceback: Traceback (most recent call last):
  File "/home/build-user/llvm-project/llvm/utils/lit/lit/TestingConfig.py", line 127, in load_from_path
    exec(compile(data, path, 'exec'), cfg_globals, None)
  File "/home/build-user/swift/test/lit.cfg", line 585, in <module>
    config.substitutions.append( ('%swift-host-lib-dir', config.swift_host_lib_dir))
AttributeError: 'TestingConfig' object has no attribute 'swift_host_lib_dir'
```

--- a/test/lit.cfg
+++ b/test/lit.cfg
@@ -582,7 +582,6 @@ else:
 
 config.substitutions.append( ('%llvm_obj_root', config.llvm_obj_root) )
 config.substitutions.append( ('%swift-lib-dir', config.swift_lib_dir) )
-config.substitutions.append( ('%swift-host-lib-dir', config.swift_host_lib_dir))
 config.substitutions.append( ('%swift-libexec-dir', config.swift_lib_dir) )
 config.substitutions.append( ('%swift-plugin-dir', config.swift_plugin_dir) )
 config.substitutions.append( ('%llvm_src_root', config.llvm_src_root) )

